spec.md

```diff
@@ -146,32 +146,48 @@ that contains both context and data).
   * REQUIRED
   * MUST be a non-empty string
 
-### source
-* Type: Object
-* Description: This describes the software instance that emits the event at
-  runtime (i.e. the producer). It contains sub-properties (listed below)
-* Constraints:
-  * REQUIRED
-  * MUST contain at least one non-empty sub-property.
-
-### source-type
+### topic
 * Type: String
-* Description: Type of the event source. Providers define list of event
-  sources.
+* Description: This expression is a classification of the event that 
+  enables groupwise publishing of and subscription to events of the 
+  described class. The classification might reflect the identity of 
+  the party raising the event, and/or the context of where and/or why 
+  the event was raised, and/or a structured classification of the 
+  information contained in the event. The value SHOULD be a URI. 
+  The value MAY be used to map directly to the "topic" concept found
+  in many middleware products. The value MAY also be a interpreted as 
+  a URL. The topic property value's scope is set by the "namespace" 
+  property. 
 * Constraints:
   * REQUIRED
-  * MUST be a non-empty string
 * Examples:
-  * s3
-
-### source-id
+  * Resource path, relative URI reference: 
+    /tenant/group/type/myresource
+  * Machine component, relative URI reference:
+    /robot/drives/3/temperature
```

Comment:
Request: add clarification for URI segments, like /namespaces/{namespaceId}/buckets/{bucketId}/objects or /robots/{robotId}/drives/{driveId}/temperature